README.md

```diff
@@ -1,6 +1,6 @@
 # Curriculum Review Tool
 
-The [Curriculum Review Tool](https://www.consumerfinance.gov/practitioner-resources/youth-financial-education/curriculum-review/tool/) is an interactive tool that allows a teacher to assess the merits of a financial education curriculum. It is an alternative to the existing [paper-based tool](https://files.consumerfinance.gov/f/documents/cfpb_youth-financial-education-curriculum-review-tool.pdf). The assessment is made across four dimensions: content, quality, utility, and efficacy. After responding to questions in each of those dimensions, users can print or save a PDF the results of the assessment.
+The [Curriculum Review Tool](https://www.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/curriculum-review/before-you-begin/) is an interactive tool that allows a teacher to assess the merits of a financial education curriculum. It is an alternative to the existing [paper-based tool](https://files.consumerfinance.gov/f/documents/cfpb_youth-financial-education-curriculum-review-tool.pdf). The assessment is made across four dimensions: content, quality, utility, and efficacy. After responding to questions in each of those dimensions, users can print or save a PDF the results of the assessment.
```

Comment:
> "save a PDF the results ..."   

Just noticed an awkward phrase. This might be better:

```suggestion
The [Curriculum Review Tool](https://www.consumerfinance.gov/consumer-tools/educator-tools/youth-financial-education/curriculum-review/before-you-begin/) is an interactive tool that allows a teacher to assess the merits of a financial education curriculum. It is an alternative to the existing [paper-based tool](https://files.consumerfinance.gov/f/documents/cfpb_youth-financial-education-curriculum-review-tool.pdf). The assessment is made across four dimensions: content, quality, utility, and efficacy. After responding to questions in each of those dimensions, users can print or save a PDF of the assessment results.
```